composer.json

```
@@ -42,7 +42,7 @@
         "drupal/asset_injector": "2.19.0",
         "drupal/auto_entitylabel": "3.0.0",
         "drupal/better_exposed_filters": "6.0.3",
-        "drupal/blazy": "2.24.0",
+        "drupal/blazy": "2.25.0",
```

Shall we update this to use [2.26](https://www.drupal.org/project/blazy/releases/8.x-2.26) (which includes "[Hotfix for QSA](https://git.drupalcode.org/project/blazy/-/compare/8.x-2.25...8.x-2.26?from_project_id=59405&straight=false)")?
```suggestion
        "drupal/blazy": "2.26.0",
```